posts/2020-08-16-ocp-log-forwarding.adoc

```diff
@@ -0,0 +1,421 @@
+---
+layout: post
+title: "Forwarding Open Liberty logs in OpenShift Container Platform to Splunk using Log Forwarding API"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/leemelim
+author_github: https://github.com/leemelim
+seo-title: Forwarding Open Liberty logs in OpenShift Container Platform to Splunk using Log Forwarding API - OpenLiberty.io
+seo-description: Using Log Forwarding API, you can send logs from Open Liberty deployed in OpenShift Container Platform to remote destinations. You can deploy Splunk on your external machine or inside your OpenShift Container Platform. The logs can be forwarded to Splunk using the Fluentd forward protocol.
+blog_description: Using Log Forwarding API, you can send logs from Open Liberty deployed in OpenShift Container Platform to remote destinations. You can deploy Splunk on your external machine or inside your OpenShift Container Platform. The logs can be forwarded to Splunk using the Fluentd forward protocol.
+---
+= Forwarding Open Liberty logs in OpenShift Container Platform to Splunk using Log Forwarding API
+Halim Lee <https://github.com/leemelim>
+
+OpenShift Container Platform provides a log aggregation solution with cluster logging using Elasticsearch, Fluentd, and Kibana (EFK stack). In addition to EFK stack, integration with other logging applications has been made available. Starting OpenShift 4.3, you can use Log Forwarding API, which is currently in Technology preview, to send logs from Open Liberty in OpenShift cluster to other available log analysis solutions.
+
+Splunk is one of the most popular logging solutions. If you already have Splunk deployed on your external machine, or if you want to start using Splunk as your new logging solution, you will have to integrate OpenShift with Splunk. Collecting logs in one location helps you monitor all of your applications efficiently, especially because container orchestration system is highly distributed and dynamic. Log Forwarding API simplifies the integration ability of OpenShift cluster with Splunk. You can configure custom pipelines to send logs to desired endpoints within or outside of your OpenShift cluster. 
```

Comment:
Splunk is one of the most popular logging solutions. 
->
Splunk is one of the most popular log analysis solutions.